apps/shell/app.js

```diff
@@ -55,9 +57,44 @@ app.use(process.env.PASSENGER_BASE_URI || '/', router);
 var server = new http.createServer(app);
 var wss = new WebSocket.Server({ noServer: true });
 
+whitelist = [];
```

Comment:
Even though you can get away with not defining the variable as `var, let, const` we want to do it to maintain consistency. 

```suggestion
let whitelist = [];
```